doc/api-extensions.md

```diff
@@ -2273,3 +2273,8 @@ This allows copying storage volume snapshots to and from remotes.
 ## `zfs_delegate`
 This implements a new `zfs.delegate` volume Boolean for volumes on a ZFS storage driver.
 When enabled and a suitable system is in use (requires ZFS 2.2 or higher), the ZFS dataset will be delegated to the container, allowing for its use through the `zfs` command line tool.
+
+## `operations_get_query_all_projects`
```

Comment:
The commit that adds the API extension should come first in the PR (so its easily identifiable when backporting in the LTS) and should be its own commit only containing the changes to doc/api-extensions.md and shared/version/api.go.

Reply:
got it. will fix this with rebase.